main.go

```diff
@@ -261,6 +265,74 @@ func writeNewXWithOptionsAndDefaults(buf *jen.File, c Config) {
 	})
 }
 
+var (
+	DebugMapFieldTag = "debugmap"
```

Comment:
nit: const

Reply:
I had it `const`, but the next var can't be, so I just combined them